examples/Datums.purs

```diff
@@ -1,6 +1,7 @@
--- | An example of fetching datums from `ogmios-datum-cache`. Helpful to test
--- | out the datum-cache integration
+-- | An example of fetching datums from `Kupo` or `Blockfrost`. Helpful to test
+-- | out the backend integration
 -- |
+-- TODO Rewrite this example or remove it
```

Comment:
@jy14898 Why rewrite?

Reply:
It assumes a certain state of the chain + datum indexer. I suppose as an example it's fine, but as a test I don't think it's great